repository/m.json

```diff
@@ -1167,6 +1167,21 @@
 				}
 			]
 		},
+		{
+		    "name": "MCPHelper",
+		    "description": "Plugin for Model Context Protocol (MCP) integration",
+		    "author": "David Donohue",
+		    "details": "https://github.com/sdirishguy/MCPHelperSublimePlugin",
+		    "labels": ["mcp", "ai", "code-generation", "sublime-text"],
+		    "releases": [
+		        {
+		            "version": "1.0.2",
+		            "url": "https://github.com/sdirishguy/MCPHelperSublimePlugin/archive/v1.0.2.zip",
```

Comment:
Remove both "version" and "url".  We want the registry crawler to find the latest tag ("tags: true") automatically.  That way you just need to push new semver tags to the public repo.  And the crawler will see them, and prepare the "zip" for you and us.

Reply:
OK I just removed them, again, sorry about that, this is my first contribution to open source...so i'm learning it as i go. Thanks for your patience.